pkgs/default.nix

```diff
@@ -68,6 +68,17 @@ lib: final: prev: with final;
     });
   });
 
+  alsa-lib = prev.alsa-lib.overrideAttrs (
+    _: _: {
+      patches = [
```

Comment:
Please use the final and previous parts to append the new patch in case `nix flake update` changes the patches being used.